File: ios/Classes/data/ServiceStatus.swift
//
//  ServiceStatus.swift
//  permission_handler
//
//  Created by Maurits van Beusekom on 12/02/2019.
//

import Foundation

enum ServiceStatus : String, Codable {
    case disabled = "disabled"
    case enabled = "enabled"
    case notApplicable = "notApplicable"
    case unknown = "unknown"
}

File: android/src/main/kotlin/com/baseflow/permissionhandler/data/ServiceStatus.kt
package com.baseflow.permissionhandler.data

import com.google.gson.annotations.SerializedName

enum class ServiceStatus {
    @SerializedName("unknown")
    UNKNOWN,
    @SerializedName("disabled")
    DISABLED,
    @SerializedName("enabled")
    ENABLED,
    @SerializedName("notApplicable")
    NOT_APPLICABLE,
}